### PR TITLE
Rename `FormView` -> `FeatureFormView`

### PR DESCRIPTION
**Pull request by mhdostal**

### CODE REVIEW

File: Examples/Examples/FormViewExampleView.swift
Comment (dfeinzimer) [inline Examples/Examples/FormViewExampleView.swift:1]: Should the name of the file become `FeatureFormViewExampleView.swift`?

File: UI Test Runner/UI Test Runner/TestViews/FormViewTestView.swift
Comment (dfeinzimer) [inline UI Test Runner/UI Test Runner/TestViews/FormViewTestView.swift:1]: `FeatureFormTestView.swift`

File: Sources/ArcGISToolkit/Components/FormView/FeatureFormView.swift
Comment (dfeinzimer) [inline Sources/ArcGISToolkit/Components/FormView/FeatureFormView.swift:1]: Folder name `.../Components/FeatureFormView/...`

**Review by dfeinzimer — COMMENTED** [review]
I think this is nearly everything, just a few things.

File: Sources/ArcGISToolkit/Components/FormView/FeatureFormView.swift
Comment (dfeinzimer) [reply Sources/ArcGISToolkit/Components/FormView/FeatureFormView.swift:1]: Ah, see you just got that one.

File: Examples/Examples/FormViewExampleView.swift
Comment (dfeinzimer) [reply Examples/Examples/FormViewExampleView.swift:1]: Or `FeatureFormExampleView.swift`

**Comment by mhdostal** [comment]
@dfeinzimer Got those two missing file renames. Thanks!

**Review by dfeinzimer — APPROVED** [review]
Builds good

**Pull request merged**